.aegir.js

```diff
@@ -4,9 +4,12 @@ const Node = require('./test/nodejs-bundle/nodejs-bundle.js')
 const PeerInfo = require('peer-info')
 const PeerId = require('peer-id')
 const pull = require('pull-stream')
+const {parallel} = require('async')
 
 const sigServer = require('libp2p-webrtc-star/src/sig-server')
+const wsRendezvous = require('libp2p-websocket-star-rendezvous')
 let server
+let server2
```

Comment:
Since we now have two "servers", let's use qualifying names:

- wrtcRendezvous
- wsRendezvous